### DISCUSSION TITLE
Class long_desc similar to class_option

### DISCUSSION THREAD

**Discussion by FilBot3** [erikhuda/thor#832] [Ideas]
Is there a way to do Class or top-level long_desc similar to each individual command? Like when someone runs your command `awesomecli help` to make that display a long description?

**Comment by wifelette** [comment]
This would be a nifty feature :) For reference, I currently accomplish this by using `print-wrapped` inside of `initialized`, and then did `super`. Like so: 

```
    def help(*)
      print_wrapped <<~INTRO
        👋 Hello friend!

        Super long message went here! 
      INTRO

      puts

      super
    end
```
